### PR TITLE
Integrate UI build and bundle tasks into Drone CI

### LINKED ISSUE #1335: Integrate UI plugin build process into CI
As the build and deploy workflow has solidified, the plugin build process needs to be integrated into CI. This will require Drone to pull Ant and two SDKs - vSphere Web Client SDK and Flex SDK to be able to compile the VIC plugin successfully. Need to confirm if it's ok to put the SDK packages in Bintray.

### PR DESCRIPTION
**Pull request by jooskim**
This PR adds a build task for VIC UI plugin that will run after a successful build, unit and integration tests. The bundle task was also slightly modified so that the tarball file contains the UI installer and compiled plugin contents.

Fixes #1335

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -73,6 +94,9 @@ build:
       - cp $BIN/vic-ui* $BIN_TEMP_DIR
       - cp $BIN/appliance.iso $BIN_TEMP_DIR
       - cp $BIN/bootstrap.iso $BIN_TEMP_DIR
+      - mkdir $BIN_TEMP_DIR/ui
+      - cp -rf ui/installer/* $BIN_TEMP_DIR/ui
+      - sleep 1
```
Comment (andrewtchin) [inline .drone.yml]: What's this for?

Comment (jooskim) [inline .drone.yml]: This is to include UI installer and plugin bundle.

Comment (andrewtchin) [inline .drone.yml]: I mean the sleep that seems unnecessary

Comment (jooskim) [inline .drone.yml]: When I tested the the UI portion on my Vagrant box there was an issue w/ tar after the 'sleep 1' command where not some files were still being copied and thus tar failed. I assume that was the VM being slow so do you think it's ok to take that out?

Comment (jooskim) [inline .drone.yml]: I'll try replace the `cp -rf ui/installer/*...` and try building it without `sleep 1`

**Comment by emlin** [comment]
lgtm

**Comment by jooskim** [comment]
@mhagen-vmware Please review the proposed commit. Thanks!

**Comment by andrewtchin** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
lgtm

please make sure to update the secrets.yml google doc as well.

**Comment by jooskim** [comment]
♻️

**Pull request merged**